### DISCUSSION TITLE
190

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#163] [General]
# 潮流周刊第190期 - 螃蟹螃蟹

封面图拍摄于在一个月前叮咚买菜上买的两只相守蟹，生命力超级旺盛，吃得很杂，蔬菜、虾米甚至干面条都吃，成为了家里两只猫的现实世界的小电视。

https://weekly.tw93.fun/posts/190-%E8%9E%83%E8%9F%B9%E8%9E%83%E8%9F%B9

**Comment by FelixLyfe** [comment]
为什么我访问不了那个 apple music 的播放列表，求教程

**Reply by FelixLyfe** [reply]
我会了，用 tw 的梯子

**Reply by la-aa** [reply]
虽然但是，还是搜不到，我是学生认证，切换tw 的节点依旧是找不到
